cliparser.py

```diff
@@ -26,7 +26,7 @@
 
 
 class CliParser(argparse.ArgumentParser):
-    def __init__(self, cli_support=None, board_names=None, add_help=True):
+    def __init__(self, cli_support=None, board_names=None, add_help=True, stack_name=None):
```

Comment:
`stack_name` is no longer necessary.

Reply:
Sorry. [removed](https://github.com/intel/auto-pts/pull/742/commits/9cc335597be5ab45e0053f491fc68571826a5372)